src/firebase/firebase-config.ts

```diff
@@ -0,0 +1,11 @@
+/** @format */
+
+export const firebaseConfig = {
```

Comment:
This seems to be duplicating code found in `src/firebase/firebase.tsx`

Reply:
Whoops, that change should not be in this set of commits. I moved the firebase Config into a different file on the branch where I'm cleaning up dependencies and warnings. Ideally we would have this in a different file and in the .gitignore so that the API key is not just out for anyone to see in source control.

I'll remove this change from this PR tho